backends/graphics/openglsdl/openglsdl-graphics.h

```diff
@@ -97,6 +101,8 @@ class OpenGLSdlGraphicsManager : public OpenGL::OpenGLGraphicsManager, public Sd
 	uint _graphicsScale;
 	bool _gotResize;
 
+	bool _lockedScreen = false; // New member variable to track lock state
```

Comment:
What makes it "New" besides being in this PR? :)
```suggestion
	bool _lockedScreen = false; // Variable to track lock state
```